castervoice/rules/core/navigation_rules/navigation_support.py

```diff
@@ -16,8 +16,11 @@ def get_direction_choice(name):
 
  # Insurers comma is recognized consistently with DNS/Natlink
  # if/else statement workaround engines that do not expect punctuation symbol as a command
-if (dragonfly.get_current_engine().name == 'natlink'):
-    comma = "(comma | ,)"
+if hasattr(dragonfly.get_current_engine(), "name"):
```

Comment:
With hasattr don't all engines have the attribute name??

Reply:
This change allowed me to step through that bit interactively (i.e. when there is no current engine). That first bit evaluates to False if interactive and True if being run with an engine